cmd/createtree/main.go

```diff
@@ -141,16 +142,28 @@ func newPK(opts *createOpts) (*any.Any, error) {
 	switch opts.privateKeyType {
 	case "PEMKeyFile":
 		if opts.pemKeyPath == "" {
-			return nil, errors.New("empty PEM path")
-		}
-		if opts.pemKeyPass == "" {
-			return nil, fmt.Errorf("empty password for PEM key file %q", opts.pemKeyPath)
+			return nil, errors.New("empty pem_key_path")
 		}
 		pemKey := &keyspb.PEMKeyFile{
 			Path:     opts.pemKeyPath,
 			Password: opts.pemKeyPass,
 		}
 		return ptypes.MarshalAny(pemKey)
+	case "PrivateKey":
+		if opts.pemKeyPath == "" {
+			return nil, errors.New("empty pem_key_path")
+		}
+		pemSigner, err := keys.NewFromPrivatePEMFile(
+			opts.pemKeyPath, opts.pemKeyPass)
```

Comment:
I think it's best to limit it to files since, as you point out, there are risks to allowing a private key to appear on the command-line. Besides, if someone doesn't want it to touch disk, they can do something like:
```shell
openssl ecparam -name secp256k1 -genkey -noout | createtree --pem_key_path /dev/stdin --pem_key_password "foo"
```